src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -0,0 +1,354 @@
+            <SplitItem isFilled>
+              <TextInput
+                value={maxSize}
+                isRequired
+                type="number"
+                id="maxSize"
+                aria-label="max size value"
+                onChange={handleMaxSizeChange}
+                min="0"
+              />
+            </SplitItem>
+            <SplitItem>
+              <FormSelect value={maxSizeUnits} onChange={handleMaxSizeUnitChange} aria-label="Max size units input">
+                <FormSelectOption key="1" value="1" label="B" />
+                <FormSelectOption key="2" value={1024} label="KiB" />
+                <FormSelectOption key="3" value={1024 * 1024} label="MiB" />
+              </FormSelect>
+            </SplitItem>
+          </Split>
+        </FormGroup>
+        <FormGroup label="Maximum age" fieldId="maxAge" helperText="The maximum age of recording data stored to disk">
+          <Split hasGutter={true}>
+            <SplitItem isFilled>
+              <TextInput
+                value={maxAge}
+                isRequired
+                type="number"
+                id="maxAgeDuration"
+                aria-label="Max age duration"
+                onChange={handleMaxAgeChange}
+                min="0"
+              />
+            </SplitItem>
+            <SplitItem>
+              <FormSelect value={maxAgeUnits} onChange={handleMaxAgeUnitChange} aria-label="Max Age units Input">
+                <FormSelectOption key="1" value="1" label="Seconds" />
+                <FormSelectOption key="2" value={60} label="Minutes" />
+                <FormSelectOption key="3" value={60 * 60} label="Hours" />
+              </FormSelect>
+            </SplitItem>
+          </Split>
+        </FormGroup>
+        <ActionGroup>
+          {props.isSettingsForm || (
+            <Button
+              variant="primary"
+              onClick={handleSubmit}
+              isDisabled={isFormInvalid || isLoading}
+              {...createButtonLoadingProps}
+            >
+              {isLoading ? 'Creating' : 'Create'}
+            </Button>
+          )}
+          <Button
+            variant={props.isSettingsForm ? 'primary' : 'secondary'}
+            onClick={handleSaveConfig}
+            isDisabled={isFormInvalid || isSaveLoading}
+            {...saveButtonLoadingProps}
+          >
+            {isSaveLoading ? 'Saving' : 'Save configuration'}
+          </Button>
+          {showHelperMessage && !isSaveLoading && (
+            <HelperText className={`${automatedAnalysisRecordingName}-config-save-helper`}>
+              <HelperTextItem variant="success">
+                <Text component={TextVariants.p}>Automated analysis recording configuration saved.</Text>
+                {!props.isSettingsForm && (
+                  <Text>
+                    You can also change this in the&nbsp;
+                    <Button isInline component="a" href="settings" variant="link" icon={<CogIcon />}>
```

Comment:
Using `href` will causes reloading of the entire application (similar to entering url on the brower). See #460 for details.

I think we should use 

```jsx
import { Link } from 'react-router-dom';

<Text>
  You can also change this in the&nbsp;
  <Button isInline component="a" variant="link" icon={<CogIcon />}>
    <Link to="/settings">Settings</Link>
  </Button>
  &nbsp;view.
</Text>
```

Reply:
Using `Link` will just switch the route instead of reloading.